Profile mode for ios devices is broken when ios language is set to swift

The profile mode for ios devices is broken when the ios language is set to swift.

## Steps to Reproduce

<!-- Please tell us exactly how to reproduce the problem you are running into. -->

1. Create a new project using swift as the ios language:
 `flutter create -i swift profile_mode_broken`
2. Sign Provisioning Profile etc.
1. Start the app in profile mode on an ios device: 
`flutter run --profile`
## Logs

<!--
      Include the full logs of the commands you are running between the lines
      with the backticks below. If you are running any "flutter" commands,
      please include the output of running them with "--verbose"; for example,
      the output of running "flutter --verbose create foo".
-->

```
flutter run --profile
Launching lib/main.dart on Chris iPhone in profile mode...
Signing iOS app for device deployment using developer identity: "iPhone Developer: xxx@t-online.de (xxx)"
Starting Xcode build...                                          
 ├─Building Dart code...                              2,7s                                                                                                                                                                                                              
 ├─Generating dSYM file...                            0,3s                                                                                                                                                                                                              
 ├─Stripping debug symbols...                         0,0s                                                                                                                                                                                                              
 ├─Assembling Flutter resources...                    1,0s                                                                                                                                                                                                              
 └─Compiling, linking and signing...                  2,6s                                                                                                                                                                                                              
Xcode build done.                                            8,0s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    === BUILD TARGET Runner OF PROJECT Runner WITH CONFIGURATION Profile ===
    /Users/chris/Development/projects/client/ios/Runner/AppDelegate.swift:10:5: error: use of unresolved identifier 'GeneratedPluginRegistrant'
        GeneratedPluginRegistrant.register(with: self)
        ^~~~~~~~~~~~~~~~~~~~~~~~~

Could not build the precompiled application for the device.

Error running application on xxx iPhone. 
```

<!-- If possible, paste the output of running `flutter doctor -v` here. -->

```
flutter doctor -v
[✓] Flutter (Channel master, v0.11.13-pre.2, on Mac OS X 10.14.1 18B75, locale de-DE)
    • Flutter version 0.11.13-pre.2 at /Users/chris/Development/flutter
    • Framework revision 72926bdff7 (3 hours ago), 2018-11-29 09:32:11 -0800
    • Engine revision 72c7a75672
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[✓] Android toolchain - develop for Android devices (Android SDK 27.0.3)
    • Android SDK at /Users/chris/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-27, build-tools 27.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    • ios-deploy 1.9.4
    • CocoaPods version 1.5.2

[✓] Android Studio (version 3.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 24.2.1
    • Dart plugin version 173.4700
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)

[✓] IntelliJ IDEA Ultimate Edition (version 2018.2.4)
    • IntelliJ at /Users/chris/Applications/JetBrains Toolbox/IntelliJ IDEA Ultimate.app
    • Flutter plugin version 28.0.4
    • Dart plugin version 182.4323.44

[✓] Connected device (2 available)
    • Chris iPhone • 662d19f06a36af09a9bd0e64f7b0fdcd83fc5167 • ios • iOS 12.1
    • iPhone 7     • 16D3476F-4685-48C1-89FF-7D3D60F61C2A     • ios • iOS 12.1 (simulator)

• No issues found!

```

```
Launching lib/main.dart on Adrian’s iPhone in profile mode...
Automatically signing iOS for device deployment using specified development team in Xcode project: G739HAJ9R6
Starting Xcode build...
Xcode build done.                                           32.7s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/Adrian/dev/buskur/buskur/ios/Runner/AppDelegate.swift:10:5: error: use of unresolved identifier 'GeneratedPluginRegistrant'
        GeneratedPluginRegistrant.register(with: self)
        ^~~~~~~~~~~~~~~~~~~~~~~~~

Could not build the precompiled application for the device.

Error running application on Adrian’s iPhone.

```

```
[✓] Flutter (Channel stable, v1.0.0, on Mac OS X 10.13.6 17G65, locale en-US)
    • Flutter version 1.0.0 at /Users/Adrian/dev/flutter
    • Framework revision 5391447fae (10 days ago), 2018-11-29 19:41:26 -0800
    • Engine revision 7375a0f414
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[✓] Android toolchain - develop for Android devices (Android SDK 27.0.3)
    • Android SDK at /Users/Adrian/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-27, build-tools 27.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    • ios-deploy 1.9.4
    • CocoaPods version 1.5.3

[✓] Android Studio (version 3.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 31.1.1
    • Dart plugin version 181.5656
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] VS Code (version 1.29.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 2.20.0

[✓] Connected device (1 available)
    • Adrian’s iPhone • 00008020-0002354A36F8002E • ios • iOS 12.1.1

• No issues found!

```

--- a/packages/flutter_tools/templates/app/ios-swift.tmpl/Runner.xcodeproj/project.pbxproj.tmpl
+++ b/packages/flutter_tools/templates/app/ios-swift.tmpl/Runner.xcodeproj/project.pbxproj.tmpl
@@ -321,6 +321,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = {{iosIdentifier}};
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_OBJC_BRIDGING_HEADER = "Runner/Runner-Bridging-Header.h";
 				SWIFT_VERSION = 4.0;
 				VERSIONING_SYSTEM = "apple-generic";
 			};

